app/dashboards/news_item_dashboard.rb

```diff
@@ -26,11 +26,9 @@ class NewsItemDashboard < Administrate::BaseDashboard
   COLLECTION_ATTRIBUTES = [
     :id,
     :title,
-    :body,
     :state,
-    :published_at,
-    :created_at,
     :updated_at,
+    :published_at,
```

Comment:
@benichu This array ends with a comma, is that normal ? Coming from you I guess it is, but I've never seen this behavior.